bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -935,8 +935,13 @@ private void updateMergeReadyComment(boolean isReady, String commitMessage, bool
                 pr.updateComment(existing.get().id(), message);
             }
         } else if (existing.isPresent()) {
-            log.info("Updating merge ready comment as no longer ready");
-            pr.updateComment(existing.get().id(), getMergeNoLongerReadyComment());
+            String noLongerReadyComment = getMergeNoLongerReadyComment();
+            if (!existing.get().body().equals(noLongerReadyComment)) {
+                log.info("Updating merge ready comment as no longer ready");
+                pr.updateComment(existing.get().id(), noLongerReadyComment);
+            } else {
+                log.info("No longer ready comment already exists, no need to update the comment");
+            }
```

Comment:
The logs should reflect that the existing comment is not equal to the new comment. Because we may change the content of the `no longer ready comment` or the `merge ready comment` in the future. Then the `pr.updateComment` is invoked, but it is not because of `Updating merge ready comment as no longer ready`.

Reply:
> The logs should reflect that the existing comment is not equal to the new comment. Because we may change the content of the `no longer ready comment` or the `merge ready comment` in the future. Then the `pr.updateComment` is invoked, but it is not because of `Updating merge ready comment as no longer ready`.

Got it! Thanks!